src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -88,6 +88,9 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	if(!path.toFile().exists()){
```

Comment:
Storage should only perform storage related actions and not be able to output to the UI.
Did not use exceptions to detect the missing file and warn the user. Please look at your peers' pull requests.